### DISCUSSION TITLE
Daily 31/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#485] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve haciendo las validaciones de los formularios de la parte de administrador. Empecé, también a hacer el property binding pero luego acordamos que se termine para el próximo sprint. Por la noche tuvimos reunión para evaluar en que condiciones estamos.
Hoy estaré terminando la parte de formularios, agregando las validaciones de casilleros. Por otro lado voy a hacer tarea de Inglés ya que tengo clases por la noche.

**Comment by rpgrca** [comment]
Recién terminé el listado simple de usuarios sin ABM. También estoy mirando el video de rest_framework que subió el profesor y acomodé un poco el back-end para que se parezca a lo que usa, tal vez para el próximo sprint habrá más información para seguir acomodando lo que tenemos. Hoy voy a limpiar un poco más la autenticación para que se desloguee cuando expira (ahora son 15 minutos, voy a subirlo a una hora tal vez), y quiero empezar con lo de los paths protegidos.

**Comment by BraianTroncoso** [comment]
Anoche y hoy estuve viendo el tema del bug, no encontraba el error. Roberto pudo resolverlo era sintaxis de json. Estoy viendo sobre back en django y profundizando en algunas cuestiones.
